Content.Server/StationEvents/Components/VentCrittersRuleComponent.cs

```diff
@@ -14,4 +14,10 @@ public sealed partial class VentCrittersRuleComponent : Component
     /// </summary>
     [DataField("specialEntries")]
     public List<EntitySpawnEntry> SpecialEntries = new();
+
+    /// <summary>
+    /// Toggle sending an announcement #IMP
+    /// </summary>
+    [DataField("announce")]
```

Comment:
no need to re-define the tag as `"announce"`, auto-generation already handles it